keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -610,16 +571,14 @@ metadata:
   name: my-svc
   namespace: my-ns
 spec:
-  ipFamily: IPv4
   ip: 42.42.42.42
+  type: "SuperclusterIP"
```

Comment:
When are we going to "fix" the name "supercluster" ?  It's long and I hate it

Reply:
Now is probably a good time to open that up, what about just "Mesh"? This would become "MeshIP" - I'm open to any ideas. A good topic for next week's sig meeting